### DISCUSSION TITLE
Locations are not received in IOS

### DISCUSSION THREAD

**Discussion by alm1989** [Almoullim/background_location#72] [General]
Thank you for all first. I'm trying to get the continuous updated locations, but only the callback of the 'getLocationUpdates' method works on Android, not on IOS. I have traced the code and the location if it is received in IOS in the locationManager, but the method SwiftBackgroundLocationPlugin.channel?.invokeMethod("location", arguments: location) of the class SwiftBackgroundLocationPlugin doesn't works. 

Can you  check that in IOS device?

Thank you again.

**Comment by manuel-junior** [comment]
I'm having the same problem. I've you find any solution?
